fastlane/lib/fastlane/fastlane_require.rb

```diff
@@ -62,6 +62,13 @@ def find_gem_name(user_supplied_name)
 
         return gems.first
       end
+
+      def format_gem_require_name(gem_name)
+        # from "fastlane-plugin-xcversion" to "fastlane/plugin/xcversion"
+        gem_name = gem_name.tr("-", "/") if gem_name.start_with? "fastlane-plugin-"
```

Comment:
Would replace "fastlane/plugin/" to "fastlane-plugin-" be more portable? E.g. if the plugin name has a dash in it?